toolboxes/nfft/gpu/ConvolverNC2C_atomic.cuh

```diff
@@ -89,7 +89,15 @@ static void NFFT_iterate_body(vector_td<unsigned int, D> matrix_size_os,
             (batch*num_frames+frame)*prod(matrix_size_os) + co_to_idx( vector_td<unsigned int, D>(grid_position), matrix_size_os );
 
         // Atomic update.
-        atomicAdd(&(image[grid_idx]), weight*sample_value);
+        if constexpr (is_complex_type_v<T>)
+        {
```

Comment:
What's wrong with the atomicAdd we already have? Also, please don't use << to divide by two. It's hacky, hard to read and (worst of all) slow.

Reply:
No issues - I just committed the bug fix from December but didn't realize you have already taken care of it as well ! This looks great - much more elegant.